ci-operator/step-registry/mco/conf/day1/tls-security-profile/mco-conf-day1-tls-security-profile-commands.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+function create_manifests(){
+    echo "Creating APIServer resource with the desired tls cofiguration in the $SHARED_DIR directory"
+
+    cat > "${SHARED_DIR}/99-apiserver.yaml" << EOF
+    apiVersion: config.openshift.io/v1
```

Comment:
We need to fix the indentation in this line and remove the fist tab/spaces so that the resulting yaml is well formed.